### PR TITLE
[vcpkg ci] Ask for adding a license field

### PR DESCRIPTION
**Pull request by autoantwort**
Fixes https://github.com/microsoft/vcpkg/discussions/20714#discussioncomment-1469700

### CODE REVIEW

**Comment by c72578** [comment]
Please amend the commit message and PR title:
Aks->Ask

**Review by Be-ing — APPROVED** [review]

**Comment by Be-ing** [comment]
Thanks for doing this.

**Comment by Be-ing** [comment]
Ping @vicroms, is there anything left to do here?

**Comment by PhoebeHui** [comment]
cc @strega-nil @strega-nil-ms

File: .github/workflows/untrustedPR.yml [hunk 1]
```diff
@@ -104,6 +106,12 @@ jobs:
               output += `The following files are affected:\n${cmake}\n`
               output += "</details>\n"
             }
+            if (missing_license !== "") {
+              output += 'You have modified or added at least one vcpkg.json where a "license" field is missing.\n'
+              output += 'If you feel able to do so, please consider adding a "license field to the following files:\n'
```
Comment (strega-nil-ms) [inline .github/workflows/untrustedPR.yml:111]:
```suggestion
              output += 'If you feel able to do so, please consider adding a "license" field to the following files:\n'

```

File: .github/workflows/untrustedPR.yml [hunk 2]
```diff
@@ -104,6 +106,12 @@ jobs:
               output += `The following files are affected:\n${cmake}\n`
               output += "</details>\n"
             }
+            if (missing_license !== "") {
+              output += 'You have modified or added at least one vcpkg.json where a "license" field is missing.\n'
+              output += 'If you feel able to do so, please consider adding a "license field to the following files:\n'
+              output += missing_license
+              output += "\nValid values for the license field are listed at https://spdx.org/licenses/\n"
```
Comment (strega-nil-ms) [inline .github/workflows/untrustedPR.yml:113]:
```suggestion
              output += "\nValid values for the license field are listed at <https://spdx.org/licenses/>\n"
```

**Review by strega-nil-ms — APPROVED** [review]
Minor typoes

**Comment by strega-nil-ms** [comment]
Thank you so much @autoantwort :)

**Pull request merged**